docs/content/docs/data-structures/sequences.md

```diff
@@ -272,6 +418,21 @@ sequences, when talking about positions in a SharedString we use the terms near
 SharedString is 0, and the farthest position is its length. When comparing two positions the nearer positions is closer
 to 0, and the farther position is closer to the length.
 
+### Examples
```

Comment:
looks like these links were stale in the source readme and now bot is picking them up.

I'm only now noticing that this section wasn't included on fluidframework.com previously. Was that intentional? It does seem useful to me fwiw.

Reply:
latest link complaint doesn't include any links in files I've edited.

Reply:
@Josmithr is working on fixing this broken link. It required changes in our fork of api-documenter.